File: common/src/main/java/net/eve0415/spigot/WebsocketIntegration/Util/WSIChatHandler.java
package net.eve0415.spigot.WebsocketIntegration.Util;

public interface WSIChatHandler {
    void chatHandler(String name, String uuid, String url, String message);

    String getName(String name, String uuid);

    static String messageFormatter(final String name, final String message) {
        return String.format("<" + name + "> " + message);
    }
}

File: common/src/main/kotlin/net/eve0415/mc/WebsocketIntegration/Interface/WSIChatHandler.kt
package net.eve0415.mc.WebsocketIntegration.Interface

public interface WSIChatHandler {
  fun chatHandler(name: String, uuid: String, url: String, message: String)

  fun getName(name: String, uuid: String): String

  fun messageFormatter(name: String, message: String): String {
    return String.format("<" + name + "> " + message)
  }
}
